modules/cpmso-yaml-failure-domain-aws.adoc

```diff
@@ -4,13 +4,18 @@
 
 :_mod-docs-content-type: REFERENCE
 [id="cpmso-yaml-failure-domain-aws_{context}"]
-= Sample AWS failure domain configuration
+= Sample {aws-short} failure domain configuration
 
-The control plane machine set concept of a failure domain is analogous to existing AWS concept of an link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-availability-zones[_Availability Zone (AZ)_]. The `ControlPlaneMachineSet` CR spreads control plane machines across multiple failure domains when possible.
+[role="_abstract"]
+To prevent downtime for your application due to the failure of a single {aws-first} region, you can configure failure domains in the control plane machine set.
+To use failure domains, you configure appropriate values in the `failureDomains` section of the `ControlPlaneMachineSet` custom resource (CR).
```

Comment:
Should this second sentence be in a separate paragraph so that it isn't in the short desc?